### DISCUSSION TITLE
Should stockholder_approval_date be nullable on StockPlan.schema.json?

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#240] [General]
Shareworks does not track plan approval by the stakeholders so this being a required field may be an issue for us.
We only have plan approval date from the company/board.

**Comment by pjohnmeyer** [comment]
TWG Notes

`stockholder_approval_date` is not required so there is no action here.
